lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -66,7 +66,16 @@ protected function transform($item) {
 
     $data = [];
 
-    $data += $this->transformSignup($item);
+    $campaign = (object) $item->campaign;
+    $current_run = $campaign->campaign_runs['current']['en']['id'];
+
+    $current = false;
+
+    if ($item->campaign_run == $current_run) {
+      $current = true;
+    }
```

Comment:
Minor, but this could just be simplified as `$current = ($item->campaign_run == $current_run);`